k8s-ci/k8s_crds/test

```diff
@@ -16,7 +16,16 @@ exit_err() {
    cat ${log};
    rm -rf ${log};
 
-   exit 1
+
+  # Snapshot of workloads
+  echo "- Snapshot of workloads on exit"
+  echo "-- Current namespace: $(current_namespace)"
+  echo "-- Pods in namespace:"
+  kubectl --namespace "$(current_namespace)" get pods || true
+  echo "-- Warning events in namespace:"
+  kubectl --namespace "$(current_namespace)" get events --field-selector type=Warning || true
+
+  exit 1
```

Comment:
nit: I could be wrong but the added line are not indented at the same level as earlier lines in that func. Either the func indent or the added line indent is wrong.

Reply:
You were right. Fixed.